libsolidity/formal/BMC.cpp

```diff
@@ -828,14 +828,14 @@ void BMC::checkCondition(
 	string extraComment = SMTEncoder::extraComment();
 	if (m_loopExecutionHappened)
 		extraComment +=
-			"\nNote that some information is erased after the execution of loops.\n"
-			"You can re-introduce information using require().";
+			"Note that some information is erased after the execution of loops.\n"
```

Comment:
Why these changes? The rest of the PR looks fine.

Reply:
yea, just an attempt to make one of the other tests work. If the tests pass here I'll remove these